sdk/azidentity/internal/cache.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package internal
+
+import "github.com/AzureAD/microsoft-authentication-library-for-go/apps/cache"
+
+// Cache represents a persistent token cache
+type Cache struct {
+	// impl is embedded as a pointer so a Cache
+	// can carry persistent state across copies
+	*impl
```

Comment:
Can this be a field? If one were to add exported methods to this type, they'd show up in public surface area which might not be immediately obvious. I don't _think_ we'd lose the state across copies unless I'm missing some subtly here.

Reply:
Sure, that works too, good idea.